lib/identity/hostdata.rb

```diff
@@ -28,7 +28,7 @@ def self.setup_settings!(configuration:, rails_env:, write_to_env: false)
     # @return [String]
     def self.domain
       @domain ||= begin
-        File.read(DOMAIN_PATH).chomp
+        File.read(File.join(root.to_s, DOMAIN_PATH)).chomp
```

Comment:
Another option would be to remove the constants (`DOMAIN_PATH`, etc) and just make them methods 🤷